en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md

```diff
@@ -0,0 +1,96 @@
+# Kafka Inbound Endpoint Example
+
+The Kafka inbound endpoint of WSO2 EI acts as a message consumer. It creates a connection to ZooKeeper and requests messages for either a topic/s or topic filters.
+
+## What you'll build
+This sample demonstrates how one way message bridging from Kafka to HTTP can be done using the inbound kafka endpoint.
+See [Configuring kafka connector](kafka-connector-configuration.md) for more information.
+
+The following diagram illustrates all the required functionality of the Kafka service that you are going to build.In this example, you only need to consider about the scenario of message consuming.
+
+<img src="/assets/img/connectors/KafkaConnector.png" title="KafkaConnector" width="800" alt="KafkaConnector"/>
```

Comment:
Shall we add a short description as well here?

Reply:
Done.